src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -61,32 +60,58 @@ public class ClusterStateMachine : RachisStateMachine
         private const string LocalNodeStateTreeName = "LocalNodeState";
         private static readonly StringSegment DatabaseName = new StringSegment("DatabaseName");
 
-        private static readonly TableSchema ItemsSchema;
-        private static readonly TableSchema CompareExchangeSchema;
+        public static readonly TableSchema ItemsSchema;
+        public static readonly TableSchema CompareExchangeSchema;
+        public static readonly TableSchema CompareExchangeTombstoneSchema;
         public static readonly TableSchema TransactionCommandsSchema;
+        public static readonly TableSchema IdentitiesSchema;
 
         public enum UniqueItems
         {
             Key,
             Index,
-            Value
+            Value,
+            PrefixIndex
         }
 
-        private static readonly Slice Items;
-        private static readonly Slice CompareExchange;
+        public enum UniqueTombstoneItems
+        {
+            Key,
+            Index,
+            PrefixIndex
+        }
+
+        public enum UniqueIdentitiesItems
+        {
+            Key,
+            Value,
+            Index,
+            KeyIndex
+        }
+
+        public static readonly Slice Items;
+        public static readonly Slice CompareExchange;
+        public static readonly Slice CompareExchangeTombstone;
         public static readonly Slice Identities;
+        public static readonly Slice IdentitiesIndexSlice;
         public static readonly Slice TransactionCommands;
         public static readonly Slice TransactionCommandsCountPerDatabase;
+        public static readonly Slice CompareExchangeIndexSlice;
+        public static readonly Slice CompareExchangeTombstoneIndexSlice;
 
         static ClusterStateMachine()
         {
             using (StorageEnvironment.GetStaticContext(out var ctx))
             {
                 Slice.From(ctx, "Items", out Items);
                 Slice.From(ctx, "CmpXchg", out CompareExchange);
+                Slice.From(ctx, "CmpXchgTombstone", out CompareExchangeTombstone);
```

Comment:
"CmpXchgTombstone" and `CompareExchangeTombstones`

Reply:
done